src/evaluator.cpp

```diff
@@ -32,16 +32,12 @@
 namespace booleval {
 
 evaluator::evaluator()
-    : is_activated_(false),
-      result_visitor_(),
-      expression_tree_()
+    : is_activated_(false)
 {}
 
 evaluator::evaluator(field_map const& fields)
-	: is_activated_(false),
-	  result_visitor_(),
-	  expression_tree_() {
-    result_visitor_.fields(fields);
+	: is_activated_(false) {
+		result_visitor_.fields(fields);
```

Comment:
Please, pay attention to indentation. It should be 4 spaces.

Reply:
Sorry Marin, Visual Studio had defaulted back to using tabs for some reason. All sorted now.